test/jdk/tools/jextract/test8254983/LibTest8254983Test.java

```diff
@@ -26,6 +26,7 @@
 import org.testng.annotations.Test;
 import static org.testng.Assert.assertEquals;
 import static test.jextract.test8254983.test8254983_h.*;
+import test.jextract.test8254983.*;
 
```

Comment:
One side effect of this non-nesting .. is that now we need two imports (import static * for functions, constants etc. and import * for structs, functional interfaces..).

Reply:
Yes - this was discussed in the mailing list. I will edit the PR comment to reflect that too, so that it becomes more evident.